### ISSUE TITLE
Dragging group nodes with children taking 100% of group size

### ISSUE THREAD

**Issue by MatheusRoichman** [xyflow/xyflow#3654]
### Describe the Bug

I'm building a WYSIWYG editor with React Flow. On this editor, we have text presets (just multiple text elements grouped) that the user can insert into the design he's building.
Some of those presets have children that fill 100% of the group size, and when I try to drag the group, it ends up moving the children inside it. I want to drag the group itself.

I tried to do it with the following implementation:

```tsx
onNodeDrag={(e, node) => {
  if (!!node.parentNode && selectedElement?.id === node.parentNode) {
    console.log(
      "Trying to drag a node that is child of the selected group"
    );

    e.stopPropagation();
  }
}}
```

Where `selectedElement` is the currently selected node. But neither `e.stopPropagation()` nor `e.preventDefault()` haven't worked.

### Your Example Website or App

https://codesandbox.io/p/sandbox/floral-sunset-h54xcx

### Steps to Reproduce the Bug or Issue

On the given example, try to move the node with ID "parent" without moving any of its children.

### Expected behavior

The group should move, and not the children.

### Platform

- OS: [e.g. macOS, Windows, Linux]: Windows
- Browser: [e.g. Chrome, Safari, Firefox]: Brave
- Version: [e.g. 91.1]: 1.60.118

### Additional context

Z-index can't be changed, because of a layers feature I have on the original code.

**Comment by moklick** [comment]
The only feature of a group is that children get positioned relatively. When you move a parent node, the child will move too. If you don't need that, you could also just render a node below the "child" node. Another thing you could try is to set `draggable: false` for each child node.

**Comment by MatheusRoichman** [comment]
Yeah, when the group moves, the children moves too, that's expected. But my problem is: when the children takes 100% of the group's size, I'm not being able to move the group. e.g., when I try to drag the group by dragging on a child, the child moves inside the group, but I want to move the group itself.

**Comment by moklick** [comment]
for that you could disable dragging for child node with the node option `draggable: false`

**Comment by MatheusRoichman** [comment]
I tried it, and I tried the "nodrag" class. On both cases, when I tried to drag, nothing happened, because the children are above the group container on the Z level, I think. And as I wrote, If the group container os 1 level above the children, I wasn't able to select a children node to edit its style.

**Comment by moklick** [comment]
I can't find a way to be able to select a node, but pass the drag handler. pointer-events:none removes the ability to select a node, draggable: false, doesn't let you drag the node below 🤔

**Comment by hiepnguyen3001** [comment]
This is also an issue for me. @moklick Is there a reason why subflow's children nodes always have the same z index as the subflow?
